Visibility updates v1.21 (#2131)
temporalio/documentation: docs-src/concepts/what-is-standard-visibility.md

---
id: what-is-standard-visibility

> title: What is standard Visibility?

Reviewer: why lowercase here?

Author: Because we're moving away from standard and advanced visibility as separate concepts (but we still support standard and advanced), this is the first step to remove it as a term from the glossary etc. 
I can revert back to cap-case but I wanted to formally deprecate it as a Temporal term.

Reviewer: 👍 there are other instances of "Standard V" if you wanted to find/replace now or at some later point

Author: Yes! Will try to catch them all in this PR.

Open Workflow Executions can be filtered by a time constraint and either a Workflow Type, Workflow Id, or Run Id.

Closed Workflow Executions can be filtered by a time constraint and either a Workflow Type, Workflow Id, Run Id, or Execution Status (Completed, Failed, Timed Out, Terminated, Canceled, or Continued-As-New).

> Support for Standard Visibility will be deprecated from Temporal Server v1.21 onwards. Check [Supported databases](/cluster-deployment-guide#supported-databases) for updates.

Reviewer 2: ```suggestion
Support for standard Visibility will be deprecated from Temporal Server v1.21 onwards. Check [Supported databases](/cluster-deployment-guide#supported-databases) for updates.
```

> Support for standard Visibility will be deprecated from Temporal Server v1.21 onwards. Check [Supported databases](/cluster-deployment-guide#supported-databases) for updates.

Reviewer 3: ```suggestion
Support for standard Visibility is deprecated beginning with Temporal Server v1.21.
For updates, check [Supported databases](/cluster-deployment-guide#supported-databases).
```
